react-instantsearch-hooks/range-slider-with-react-spectrum/src/App.js

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useState } from 'react';
+import {
+  Hits,
+  InstantSearch,
+  Pagination,
+  RefinementList,
+  SearchBox,
+  useConnector,
+} from 'react-instantsearch-hooks-web';
+import connectRange from 'instantsearch.js/es/connectors/range/connectRange';
+import { RangeSlider as SpectrumRangeSlider } from '@adobe/react-spectrum';
+
+import algoliasearch from 'algoliasearch/lite';
+
+import './App.css';
+
+const searchClient = algoliasearch(
+  'B1G2GM9NG0',
+  'aadef574be1f9252bb48d4ea09b5cfe5'
+);
+
+function App() {
+  return (
+    <div className="container">
+      <InstantSearch
+        searchClient={searchClient}
+        indexName="demo_ecommerce"
+        insights={true}
+      >
+        <div className="search-panel">
+          <div className="search-panel__filters">
+            <RefinementList attribute="brand" />
+            <RangeSlider attribute="price" />
+          </div>
+
+          <div className="search-panel__results">
+            <SearchBox className="searchbox" placeholder="Search" />
+            <Hits />
+
+            <div className="pagination">
+              <Pagination />
+            </div>
+          </div>
+        </div>
+      </InstantSearch>
+    </div>
+  );
+}
+
+function useRangeSlider(props) {
+  return useConnector(connectRange, props);
+}
+
+function RangeSlider(props) {
```

Comment:
maybe a separate file

Reply:
Addressed in [4a6d364](https://github.com/algolia/doc-code-samples/pull/457/commits/4a6d36485ab15cd3ac4046304d1b7791d5795eef).